src/cart/crt_group.c

```diff
@@ -1556,6 +1563,9 @@ crt_hdlr_uri_lookup(crt_rpc_t *rpc_req)
 	ul_out->ul_uri = cached_uri;
 	if (ul_out->ul_uri == NULL)
 		D_GOTO(out, rc = -DER_OOG);
+	if (crt_gdata.cg_use_sensors)
+		(void)d_tm_increment_counter(&crt_gdata.cg_uri_fwd, 1,
+					     NULL);
```

Comment:
i think this whole block is unreachable code now; i dont believe there is any way to get into forwarding code. we populate uris for all tags of the server once we know 1 tag;
similarly requestor/client should be querying tag0 uri when it doesnt know the rank and if uri is not found on server it should bail out with -DER_OOG on line 1555

Reply:
ok, let me remove this metric then.